crates/uniswap-v4/src/uniswap/loaders/mod.rs

```diff
@@ -1,4 +1,24 @@
 #[rustfmt::skip]
+pub mod get_uniswap_v_3_pool_data {
+    alloy::sol!(
+        #[allow(missing_docs)]
+        #[sol(rpc)]
+        #[derive(Debug, PartialEq, Eq,Hash, serde::Serialize, serde::Deserialize)]
+        GetUniswapV3PoolData,
+        "../../contracts/out/GetUniswapV3PoolData.sol/GetUniswapV3PoolData.json"
+    );
+}
+#[rustfmt::skip]
+pub mod get_uniswap_v_3_tick_data {
+    alloy::sol!(
+        #[allow(missing_docs)]
+        #[sol(rpc)]
+        #[derive(Debug, PartialEq, Eq,Hash, serde::Serialize, serde::Deserialize)]
+        GetUniswapV3TickData,
+        "../../contracts/out/GetUniswapV3TickData.sol/GetUniswapV3TickData.json"
+    );
+}
```

Comment:
we deleted the build script but keeping the bindings. any one new who clones will have this fail